delivery/repositories.py

```diff
@@ -0,0 +1,46 @@
+from abc import ABC, abstractmethod
+from delivery.error_handling import WrongIdError
+
+
+class AbstractRestaurantRepository(ABC):
+    @abstractmethod
+    def create(self, content):
```

Comment:
repository should work with data with is defined in `models`.
e.g.
```python
def update(self, restaurant: Restaurant, restaurant_id):
        raise NotImplementedError

def get_all(self) -> List[Restaurant]:
        raise NotImplementedError
```

Reply:
fixed